iib/web/api_v1.py

```diff
@@ -433,6 +433,9 @@ def patch_request(request_id):
             bundle_img = Image.get_or_create(bundle)
             bundle_img.operator = operator_img
 
+    if 'distribution_scope' in payload:
+        setattr(request, 'distribution_scope', payload['distribution_scope'])
```

Comment:
Why are you using `setattr`?
```suggestion
        request.distribution_scope = payload['distribution_scope']
```

Reply:
you're right